concepticondata/conceptlists/Carvalho-2021-206.tsv

```diff
@@ -0,0 +1,207 @@
+Carvalho-2021-206-18	18	blood	946	BLOOD
+Carvalho-2021-206-19	19	flesh	2259	FLESH
+Carvalho-2021-206-20	20	Veins	1924	VEIN
+Carvalho-2021-206-21	21	buttocks	834	BUTTOCKS
+Carvalho-2021-206-22	22	leg	1297	LEG
+Carvalho-2021-206-23	23	thigh	800	THIGH
+Carvalho-2021-206-24	24	knee	1371	KNEE
+Carvalho-2021-206-25	25	shin	186	SHIN
+Carvalho-2021-206-26	26	foot	1301	FOOT
+Carvalho-2021-206-27	27	arm	1673	ARM
+Carvalho-2021-206-28	28	elbow	981	ELBOW
+Carvalho-2021-206-29	29	hand	1277	HAND
+Carvalho-2021-206-30	30	neck	1333	NECK
+Carvalho-2021-206-31	31	skin	763	SKIN
+Carvalho-2021-206-32	32	navel	1838	NAVEL
+Carvalho-2021-206-33	33	tail	1220	TAIL
+Carvalho-2021-206-34	34	wing	1257	WING
+Carvalho-2021-206-35	35	egg	744	EGG
+Carvalho-2021-206-36	36	feces	675	FAECES (EXCREMENT)
+Carvalho-2021-206-37	37	fart	1159	FART
+Carvalho-2021-206-38	38	urine	1142	URINE
+Carvalho-2021-206-39	39	fat	323	FAT (ORGANIC SUBSTANCE)
+Carvalho-2021-206-40	40	name	1405	NAME
+Carvalho-2021-206-41	41	house	1252	HOUSE
+Carvalho-2021-206-42	42	village	930	VILLAGE
+Carvalho-2021-206-43	43	nest	539	NEST
+Carvalho-2021-206-44	44	path	2252	PATH
+Carvalho-2021-206-45	45	path2	2252	PATH
+Carvalho-2021-206-46	46	bow	994	BOW
+Carvalho-2021-206-47	47	arrow	977	ARROW
+Carvalho-2021-206-48	48	rope	1218	ROPE
+Carvalho-2021-206-49	49	canoe	1970	CANOE
+Carvalho-2021-206-50	50	raft	1664	RAFT
+Carvalho-2021-206-51	51	person	683	PERSON
+Carvalho-2021-206-52	52	spirit/soul	811	SOUL
+Carvalho-2021-206-53	53	man	1554	MAN
+Carvalho-2021-206-54	54	woman	962	WOMAN
+Carvalho-2021-206-55	55	father	1217	FATHER
+Carvalho-2021-206-56	56	father2	1217	FATHER
+Carvalho-2021-206-57	57	mother	1216	MOTHER
+Carvalho-2021-206-58	58	mother2	1216	MOTHER
+Carvalho-2021-206-59	59	son	1620	SON
+Carvalho-2021-206-60	60	daughter	1357	DAUGHTER
+Carvalho-2021-206-61	61	grandson	1618	GRANDSON
+Carvalho-2021-206-62	62	granddaughter	1619	GRANDDAUGHTER
+Carvalho-2021-206-63	63	father-in-law	1055	FATHER-IN-LAW
+Carvalho-2021-206-64	64	mother-in-law	1050	MOTHER-IN-LAW
+Carvalho-2021-206-65	65	nephew/niece	3890	NEPHEW OR NIECE
+Carvalho-2021-206-66	66	younger brother	1760	YOUNGER BROTHER
+Carvalho-2021-206-67	67	younger sister	1761	YOUNGER SISTER
+Carvalho-2021-206-68	68	grandmother	1496	GRANDMOTHER
+Carvalho-2021-206-69	69	grandfather	1383	GRANDFATHER
+Carvalho-2021-206-70	70	pet	3780	PET
+Carvalho-2021-206-71	71	paca (rodent sp.)	316	PACA
+Carvalho-2021-206-72	72	capybara	314	CAPYBARA
+Carvalho-2021-206-73	73	rat	1490	RAT
+Carvalho-2021-206-74	74	agouti	313	AGOUTI
+Carvalho-2021-206-75	75	coati	2389	COATI (COATIMUNDI)
+Carvalho-2021-206-76	76	peccary (sp.)1	2390	PECCARY
+Carvalho-2021-206-77	77	peccary (sp.)2	2390	PECCARY
+Carvalho-2021-206-78	78	anteater	181	ANTEATER
+Carvalho-2021-206-79	79	tapir	312	TAPIR
+Carvalho-2021-206-80	80	Capuchin monkey		
+Carvalho-2021-206-81	81	howler monkey>	433	HOWLER MONKEY
+Carvalho-2021-206-82	82	sloth	2446	SLOTH
+Carvalho-2021-206-83	83	snake	730	SNAKE
+Carvalho-2021-206-84	84	boa		
```

Comment:
Don't we have Boa as a concept? If not, would it make sense to add it? I mean, fi there are otehr lists?

Reply:
Hm, we have currently ANACONDA (WATER BOA) and no lists with the concept 'boa' (or 'boa constrictor') yet.